build-se

```diff
@@ -5,8 +5,8 @@ phpize && \
 make && \
 make install && \
 echo "extension=sweph.so" > /usr/local/etc/php/conf.d/sweph.ini &&
+yes n | make test && \
```

Comment:
Automatically run tests on each build and do not ask to report results to the PHP community.